### ISSUE TITLE
[3.x] Localization of phone field with country select not possible

### ISSUE THREAD

**Issue by vandres** [verbb/formie#2042]
### Describe the bug

We would like to use the phone input and allow the country selection as offered. Our website is in German and English and we create a form per language. 

I think the localization of that field is missing at the moment. All texts are in English and not changable.

Docs:

https://intl-tel-input.com/examples/localise-countries.html

### Steps to reproduce

1. Create a form with a phone field and country selection
2. open the field on a German site
3. all interface texts are in English

### Form settings

- Multi-page form: Yes
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates:  No

### Craft CMS version

5.3.x

### Plugin version

3.x

### Multi-site?

"Yes"

**Comment by engram-design** [comment]
My only issue with using the [intl-tel-input](https://intl-tel-input.com/) internationalisation is that languages are tree-shaken, and that in order to make the given language dynamic, we would need to load in every language translation into the front-end JS for Formie so that they can be switched to after build time.

This makes the `phone-country.js` filesize go from `39kb` to `194kb` (minified) which isn't great by any means.

I'll have to have a look at some more advanced options, possibly creating our own translations somehow.

**Comment by vandres** [comment]
It would be great anyway, to be able to overwrite those translations. Especially the interface texts...

**Comment by engram-design** [comment]
If anything, it'll probably be the goal to use Craft's/Formie's static translations which are probably already ready for things like country names. It'll just be figuring out how to supply those to a JS component we don't have control over!

**Comment by vandres** [comment]
In the end, the translations are just JS objects: https://github.com/jackocnr/intl-tel-input/blob/master/src/js/intl-tel-input/i18n/de/interface.ts

**Comment by engram-design** [comment]
Yep, aware of that. We'll have to construct our own for a specific language (the one you configure the field to use), rather than bundle them all at the JS build step.

**Comment by mnlmaier** [comment]
@engram-design — Do you have any plans to implement this functionality? We'd love to have the Country Names in German :)

**Comment by engram-design** [comment]
Still looking into options for this, sorry!

**Comment by engram-design** [comment]
Updated in [4.0.0-beta.10](https://github.com/verbb/formie/releases/tag/4.0.0-beta.10).
